app/app.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const { getEndpoints } = require("./controller");
+const app = express();
+
+app.use(express.json());
```

Comment:
We should only ever write code we need - we don't need this here yet, please remove